spec/site_prism/element_spec.rb

```diff
@@ -31,6 +31,16 @@
         expect(subject).not_to have_element_two
       end
 
+      it 'raises a warning when the name starts with no_' do
+        log_messages = capture_stdout do
+          SitePrism.log_level = :WARN
+          expect { subject.no_such_element }
+          .to raise_error(Capybara::ElementNotFound)
```

Comment:
I wasn't sure whether this was the best way to do this.

Reply:
This is one way to do it, however it's adding an extra test we don't need. Instead you could just do `subject.no_such_element rescue Capybara::ElementNotFound`

You can put this on one line and add a comment above `# swallow the error generated by the element not existing`